src/const/urls.ts

```diff
@@ -24,7 +24,7 @@ export const JUMPER_TX_PATH = '/tx';
 export const JUMPER_WALLET_PATH = '/wallet';
 export const JUMPER_QUESTS_PATH = '/quests';
 export const JUMPER_CAMPAIGN_PATH = '/campaign';
-export const JUMPER_LEADERBOARD_PATH = '/leaderboard';
+export const    JUMPER_LEADERBOARD_PATH = '/leaderboard';
```

Comment:
I created this issue on purpose so we can see the comment left by the CI lint-staged.